Import IconProps from @sumup/icons (#1271)
sumup-oss/circuit-ui: .changeset/nine-items-reply.md

---
'@sumup/circuit-ui': patch
---

> Fixed the import of IconProps. This could have caused TypeScript build failures.

Author: ```suggestion
Fixed the import of `IconProps` in the `IconButton` component. This was causing TypeScript build failures.
```
